src/Raven.Server/Documents/Replication/OutgoingReplicationHandler.cs

```diff
@@ -1238,8 +1238,11 @@ private void OnTimeSeriesChange(TimeSeriesChange change)
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         private void OnChangeInternal(bool triggeredByReplicationThread)
         {
-            if (triggeredByReplicationThread)
+            if (triggeredByReplicationThread &&
+                (ReplicationType != ReplicationLatestEtagRequest.ReplicationType.External ||
```

Comment:
let's be more explicit for which type of replication we do want this optimization.
I think _only_ for internal..

Reply:
done